webapp/components/wpt-results.html

```diff
@@ -372,13 +377,13 @@
       }
 
       async fetchManifest(path) {
-        if (!this.computePathIsATestFile(path)) {
-          return;
-        }
-        this.manifest = null; // Clear ASAP.
         this.manifest = await fetch(`/api/manifest?path=${path}`).then(r => r.json());
       }
 
+      computeManifest(manifests, path) {
```

Comment:
Looks like `computeManifest` is unused. In case it isn't: Is the plural `manifests` here a typo? Or is it slicing things out of a bunch of manifests, one per run?

Reply:
I was toying with keeping the manifests when changing paths, but decided that was overly complex.